src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypingEngine.java

```diff
@@ -249,6 +234,7 @@ protected VariantCallContext calculateGenotypes(final FeatureContext features,
 
         final OutputAlleleSubset outputAlternativeAlleles = calculateOutputAlleleSubset(AFresult);
 
+        // posterior probability that at least one alt allele exists in the samples
         final double PoFGT0 = Math.pow(10, AFresult.getLog10PosteriorOfAFGT0());
```

Comment:
Consider renaming this variable (`PoFGT0`) to something more human-readable/comprehensible like `postProbOfOneAltAllele`, assuming `PoFGT0` is not a universally-recognized standard abbreviation that I am just ignorant of :)